plugins/uefi/fu-plugin-uefi.c

```diff
@@ -439,6 +440,12 @@ fu_plugin_uefi_load_config (FuPlugin *plugin, FuDevice *device)
 	fu_device_set_metadata_boolean (device,
 					"RequireShimForSecureBoot",
 					!disable_shim);
+
+	/* check if using UEFI removeable path */
+	enable_removable_path = fu_plugin_get_config_value_boolean (plugin, "EnableUefiRemovablePath");
```

Comment:
if there isn't a fallback, this should rename to `UseUefiRemovablePath`

Reply:
just need to clarify the first comment I left and fix this.